roles/analysis/defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 # defaults file for analysis
 
 analysis_convert2rhel_preconv_opts: ""
+# If you are converting to RHEL 8.8 and have an Extended Upgrade Support (EUS), add the --eus option by setting this variable to true.
+analysis_convert2rhel_eus: false
```

Comment:
I am just thinking if this simply can't be added in the `analysis_convert2rhel_preconv_opts` string, do we need separate variable? I mean if you need to customize the convert2rhel options I htink only one could be used and README of roleshould mention the eus option as one of the options .. at least I think that how leapp uses is (like in https://github.com/redhat-cop/infra.leapp/blob/main/roles/upgrade/README.md )

Reply:
Yeah, so, the eus option is only necessary if we are dealing with systems that are 8.8+, for example. If we are dealing with anything that is below that, convert2rhel will try to automatically detect if the system is in EUS or not. 

The option is there because we want the user to tell us if they want their systems to land and stay in EUS. Otherwise, if they don't specify this, we will just do a normal conversion. 

Regarding the README, yes, you're correct. We will need to update it as well.

Reply:
Yeah I know why is it there, I was just trying to make the point that leapp simplified it little bit and uses string for all options, instead of modifying/having variable per every option (in this case bool), user would just add the option to the `analysis_convert2rhel_preconv_opts` string

but both are fine, I can't guess how many such options are there, maybe leapp has too many and it was easier

Reply:
I was thinking from the user perspective that it could be better to have it like this, but either just put it into `analysis_convert2rhel_preconv_opts` is okay for me.

Reply:
I would leave this the way it is right now. Seems like a better experience if we have it as a boolean variable for now.